src/visualizations.jl

```diff
@@ -58,12 +59,13 @@ See also [`abmvideo`](@ref) and [`abmexploration`](@ref).
   placed naturally.
 * `static_preplot!` : A function `f(ax, model)` that plots something after the heatmap
   but before the agents.
-* `osmkwargs = NamedTuple()` : keywords directly passed to `OSMMakie.osmplot!`
-  if model space is `OpenStreetMapSpace`.
-* `graphplotkwargs = NamedTuple()` : keywords directly passed to
-  [`GraphMakie.graphplot!`](https://graph.makie.org/stable/#GraphMakie.graphplot)
+* `spaceplotkwargs = NamedTuple()` : keywords used in `spaceplot!`. Directly passed to
```

Comment:
Similarly to the above comment, `spaceplot!` is referred to here but is not explained or defined anywhere above. So we can say something like "keywords utilized when plotting the space.". The next sentences explain where they are used in more detail.

Reply:
Fixed.